### PR TITLE
docs: add a monitoring section to the document

### PR DESCRIPTION
**Pull request by anupriya0703**
This PR adds information about Prometheus monitoring stack.

Signed-off-by: anupriya0703 <anupriya.gupta@mayadata.io>

### CODE REVIEW

File: reference/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+{% tab title="Example metrics" %}
+```text
+# HELP disk_pool_status mayastor name status
+# TYPE disk_pool_status gauge
+disk_pool_status{node="worker-0",name="mayastor-disk-pool"} 1
+# HELP disk_pool_total_size_bytes mayastor name total size in bytes
+# TYPE disk_pool_total_size_bytes gauge
+disk_pool_total_size_bytes{node="worker-0",name="mayastor-disk-pool"} 5.360320512e+09
+# HELP disk_pool_used_size_bytes mayastor name used size in bytes
```
Comment (avishnu) [inline reference/monitoring.md]: What does "mayastor name" imply? Did you mean mayastor disk pool name?

File: reference/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  selector:
+    matchLabels:
+      app: mayastor
+  endpoints:
+  - port: metrics
+```
+{% endtab %}
+
+{% hint style="info" %}
+Upon successful integration of the exporter with the Prometheus stack, the metrics will be available on the port 9090 and all the endpoints will be able to make HTTP calls.
```
Comment (avishnu) [inline reference/monitoring.md]: What does "and all the endpoints will be able to make HTTP calls" imply?

**Comment by avishnu** [comment]
Shouldn't the CSI metrics also be a part of this section?

File: reference/monitoring.md (hunk 1 above)
Comment (anupriya0703) [reply reference/monitoring.md]: I have taken this example output from here: [link](https://dcsw-bolt.atlassian.net/wiki/spaces/MS/pages/4919214/Pool+metrics+exporter#Example-of-the-above-mentioned-metrics%3A)

File: reference/monitoring.md (hunk 2 above)
Comment (anupriya0703) [reply reference/monitoring.md]: The endpoints like "/metrics" will be able to make HTTP request to retrieve the corresponding metrics.

File: reference/monitoring.md (hunk 1 above)
Comment (avishnu) [reply reference/monitoring.md]: Can you once check for cStor or any other engine of OpenEBS?

File: reference/monitoring.md (hunk 2 above)
Comment (avishnu) [reply reference/monitoring.md]: Probably, we can just stop at "on the port 9090 and HTTP endpoint /metrics."

Comment (anupriya0703) [reply reference/monitoring.md]: Updated the text.

File: reference/monitoring.md (hunk 1 above)
Comment (anupriya0703) [reply reference/monitoring.md]: updated the text.

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**